_includes/newsletter.html

```diff
@@ -5,7 +5,7 @@
   class="embeddable-buttondown-form">
   <h3>Subscribe to be notified about new posts</h3>
   <label for="bd-email">Email</label>
-  <input id="newsletter-email" type="email" name="email" id="bd-email" />
```

Comment:
Looks like there's CSS for this ID: https://github.com/Homebrew/brew.sh/blob/2d8be41b4fd2c5e9aabd73bbb61252ca48ee50af/assets/css/style.scss#L585